test/celeritas/em/Wentzel.test.cc

```diff
@@ -212,6 +212,71 @@ TEST_F(CoulombScatteringTest, mott_xs)
     EXPECT_VEC_SOFT_EQ(xsecs, expected_xsecs);
 }
 
+TEST_F(CoulombScatteringTest, wokvi_transport_xs)
+{
+    // Copper
+    MaterialId mat_id{0};
+    ElementId elm_id{0};
+
+    AtomicNumber const z = this->material_params()->get(elm_id).atomic_number();
+
+    // Incident particle energy cutoff
+    MevEnergy const cutoff = this->cutoff_params()->get(mat_id).energy(
+        this->particle_track().particle_id());
+
+    std::vector<real_type> xs;
+    for (real_type energy : {50, 100, 200, 1000, 13000})
```

Comment:
Is the test below 100MeV (Wenzel low limit) meaningful? Okay for testing as it should be a soft-limit. Is 13GeV special energy point? May add at least 100GeV and 1TeV?

Reply:
I just used the same energies as the test for the Wentzel helper, so I'm not actually sure if there is a special meaning. I'll add some higher energy points.